test/serverside.js

```diff
@@ -1496,9 +1502,11 @@ describe('App configs', function() {
 			expect(response.app.custom_action_handler_url).to.be.eq('');
 		});
 
-		// Reset custom command endpoint url to original
-		await client.updateAppSettings({
-			custom_action_handler_url: originalUrl,
+		before(async function() {
```

Comment:
Isn't this run `after` instead of `before`? Maybe we can move this block up to after the `before`?

Reply:
you are totally right, I'm rushing 🤦